tornettools/generate.py

```diff
@@ -265,7 +265,7 @@ def __server(args, network, server):
     if server['is_hs_server']:
         # this is an onion service, so tgen should only listen on localhost so that we catch errors if we accidentally
         # try connecting to the onion service from "exit" clients
-        process["environment"] = "TGENIP=127.0.0.1"
+        process["environment"] = {"TGENIP": "127.0.0.1"}
```

Comment:
Comment to block merge - maybe wait for tornettools to get tagged, though OTOH I guess if this is blocking anything we're already planning to tag  before the current HEAD so :shrug:

Reply:
There has now been a new release tagged.